ts/screens/onboarding/FingerprintScreen.tsx

```diff
@@ -0,0 +1,119 @@
+import { Button, Content, Text, View } from "native-base";
+import * as React from "react";
+
+import { NavigationScreenProps } from "react-navigation";
+
+import { connect } from "react-redux";
+
+import AbortOnboardingModal from "../../components/AbortOnboardingModal";
+import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
+
+import I18n from "../../i18n";
+import {
+  abortOnboarding,
+  fingerprintAcknowledge
+} from "../../store/actions/onboarding";
+
+import { BiometrySimpleType } from "../../sagas/startup/checkSupportedFingerprintSaga";
+import { Dispatch } from "../../store/actions/types";
+
+type FingerprintScreenNavigationParams = {
```

Comment:
```suggestion
type NavigationParams = {
```

it's ok since the type is not exported and naming is quite specific

Reply:
Fixed